src/assets/default.scss

```diff
@@ -201,7 +196,7 @@ a {
 
 .cont {
   background-image: linear-gradient(to right, $deep-purple-grad, $light-purple-grad),
-  url("https://res.cloudinary.com/jessam/image/upload/v1554725326/tachina-lee-42980-unsplash_1.jpg"); // sass-lint:disable-line no-url-domains, no-url-protocols
+  url('https://res.cloudinary.com/jessam/image/upload/v1554725326/tachina-lee-42980-unsplash_1.jpg'); // sass-lint:disable-line no-url-domains, no-url-protocols
```

Comment:
@olajide1234 why are we switching to single quotes?

Reply:
As communicated by Kinglsey, we needed to align Eslint and hound rules.